server/server.go

```diff
@@ -29,14 +29,18 @@ type Server struct {
 	db   *db.DB
 	echo *echo.Echo
 
-	verbose bool
+	verbose     bool
+	faviconPath string
 }
 
 // New creates a new server.
 func New(db *db.DB, opts ...Opt) (*Server, error) {
 	s := &Server{
 		db:   db,
 		echo: echo.New(),
+
+		verbose:     false,
+		faviconPath: "",
```

Comment:
These aren't strictly necessary, since they'll get zeroed on initialization. I'm not against adding them anyway though.

Reply:
I just wanted to be as explicit as possible :D